### ISSUE TITLE
[Request] html capable labels when addField

### ISSUE THREAD

**Issue by mcbmcb0** [kevinchappell/formBuilder#350]
I have a form editor which saves and re-loads form elements via ajax. (its awesome!)
When I create an element and write html into its label  (eg a header- `"Answer <b>all</b> questions"`) and close the edit element its displays as html fine in the label. all good!

But when I save the field (from form.data) via ajax and later reload it via ajax and using:
`formBuilder.actions.addField(data, undefined);`
then the input in the edit section shows as html entities ( eg `of &lt;b&gt;`) and the labels (eg `<label class="field-label">` show `<b>..` rather than displaying as html. And on my next save I get entites in the form data (ie not the tags). I have checked the ajax and its html (not html entities) coming back from the ajax tho it is escaped having come out a a db and sent via ajax - eg in console: `<b\/>`).
The only exception I see is for paragraphs - i guess as they use a textarea. those work great for my purposes (paste in html, have it show, save via ajax, later be re-loadable and re-editable).

sorry i can't think how to set up a fiddle for this easily, but I think its really about how `.addField` (or prepFieldVars()?) adds labels to text inputs vs textareas. I also can't figure out the js - not my strongest

**The request**: could addField insert html labels as html, not as entities? then re-editable forms could include html in various labels, not just paragraphs.

Thanks for considering this (I hope it made sense)

**Comment by mcbmcb0** [comment]
ok so this relates to escapeAttrs() which intentionally escapes markup rather than renders it. But i can't quite see why its required nor why textArea is excluded....

**Comment by kevinchappell** [comment]
@mcbmcb0 Makes perfect sense. When you get a chance have a look at the #246 . This may solve the issue for now but there are major changes coming to how html in labels and text-area are handled.

**Comment by mcbmcb0** [comment]
yep, #246 is it... looking forward to the changes you mention.
(tho as soon as you implement contenteditable  divs with markdown then users will want a ck-editor on each label!)
BTW great work on this form builder- it really is very useful
